tests/splunk_json_to_stix/test_class.py

```diff
@@ -362,4 +368,54 @@ def test_email_cim_to_stix(self):
         assert(addr_obj.keys() == {'type', 'value'})
         assert(addr_obj['type'] == 'email-addr')
         assert(addr_obj['value'] == src_user)
-        
\ No newline at end of file
+    
+    def test_custom_mapping(self):
+
+        data_source = "{\"type\": \"identity\", \"id\": \"identity--3532c56d-ea72-48be-a2ad-1a53f4c9c6d3\", \"name\": \"Splunk\", \"identity_class\": \"events\"}"
+        data = "[{\"tag\":\"network\", \"src_ip\": \"127.0.0.1\"}]"
+
+        options = {
```

Comment:
Judging from json_to_stix.py, wouldn't this need to come in as options = {'mapping': {<mapping json>}} ?

Reply:
we're using cim_to_stix.py which is inside Splunk module folder, however, I have changed it to take from mapping object inside options.